src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1785,6 +1816,24 @@ public function setIgnoreMandatoryFlag($ignoreMandatoryFlag)
     }
 
     /**
+     * Orders a child before another child
+     * @param NodeInterface $parent
+     * @param NodeInterface $childSource
+     * @param NodeInterface $childDest
+     */
+    private function executeOrderBefore(
+        NodeInterface $parent,
+        NodeInterface $childSource,
+        NodeInterface $childDest
+    ) {
+        // reorder
+        $parent->orderBefore($childSource->getName(), $childDest->getName());
+
+        $event = new ContentOrderBeforeEvent($childDest, $childSource);
+        $this->eventDispatcher->dispatch(ContentEvents::NODE_ORDER_BEFORE, $event);
```

Comment:
As you have already discovered we need this event, so I would agree with you. If it is easily possible I would also rename it to `ContentOrderEvent`, which then contains the current node, the from position and the position it has been moved to. Could be handy, and we don't know what this event will be used in the future for.